src/main/java/duke/task/ToDos.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+/**
+ * Represents a ToDo task. A ToDo object contains the description of task.
+ */
+public class ToDos extends Task {
```

Comment:
Should the name of the class be singular instead of plural? Conversely, if you would like to standardise the classes, you might want to change the Deadline class to its plural form as well.